azure-pipelines-ci.yml

```diff
@@ -67,50 +42,21 @@ jobs:
       linux-i386-gcc-6: {}
 
   steps:
-  # Rebase to origin/master for every PR. This means users don't have to
-  # rebase every time master changes. As long as the PR applies cleanly, we
-  # will validate it.
-  - script: |
-      git config user.email 'info@openttd.org'
-      git config user.name 'OpenTTD CI'
-      git rebase origin/master
-    displayName: 'Rebase to master'
-  - task: Docker@1
-    displayName: 'Build and test'
-    inputs:
-      command: 'Run an image'
-      imageName: openttd/compile-farm-ci:$(Agent.JobName)
-      volumes: '$(Build.SourcesDirectory):$(Build.SourcesDirectory)'
-      workingDirectory: '$(Build.SourcesDirectory)'
-      runInBackground: false
+  - template: azure-pipelines/ci-git-rebase.yml
+  - template: azure-pipelines/linux-build.yml
```

Comment:
is there a particular reason why the linux build doesn't require opengfx?

Reply:
The Docker images already have that installed. Basically, Linux is done via dockers, the other two are done manually. A bit annoying :D